### PR TITLE
add Sliderule applications for AGU2023

### PR DESCRIPTION
**Pull request by tsutterley**
todo: add pip magics to install latest sliderule version when made available.

### CODE REVIEW

File: book/tutorials/SlideRule_applications/SlideRule_applications.ipynb [hunk 1]
```diff
@@ -0,0 +1,1209 @@
...
+    "\n",
+    "```{admonition}Learning Objectives\n",
+    "- How to use Sliderule software for scalable ICESat-2 data access & on-demand processing\n",
+    "- Integrating ICESat-2 data with other datasets (such as raster DEMs)\n",
+    "- Visualizing ICESat-2 elevations with interactive maps and matplotlib figures\n",
+    "```\n",
+    "\n",
+    "### SlideRule Introduction\n",
+    "\n",
+    "SlideRule is a collaborative effort between NASA Goddard Space Flight Center (GSFC) and the University of Washington. It is an on-demand science data processing service that runs in on Amazon Web Services (AWS) and responds to REST API calls to process and return science results.  This new _science data as a service_ model is a new way for researchers to work and analyze data. SlideRule was designed to enable researchers to have low-latency access to custom-generated, high-level data products.\n",
```
[the remark this answers is not in the document]
Comment (tsutterley) [reply book/tutorials/SlideRule_applications/SlideRule_applications.ipynb:19]: 😳🫣

File: book/tutorials/SlideRule_applications/SlideRule_applications.ipynb [hunk 2]
```diff
@@ -0,0 +1,1209 @@
...
+    "Interactive maps within the SlideRule python API are build upon [ipyleaflet](https://ipyleaflet.readthedocs.io), which are Jupyter and python bindings for the fantastic [Leaflet](https://leafletjs.com/) javascript library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create ipyleaflet map in specified projection\n",
```
Comment (tsnow03) [inline book/tutorials/SlideRule_applications/SlideRule_applications.ipynb:166]: Should this be rendered? I don't think we have it set to render for you.

File: book/tutorials/SlideRule_applications/SlideRule_applications.ipynb [hunk 3]
```diff
@@ -0,0 +1,1209 @@
...
+    "# concatenate the results into a single geodataframe\n",
+    "gdf1 = gpd.pd.concat(elevations)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Review GeoDataFrame output\n",
+    "Can inspect the columns, number of returns and returns at the top of the GeoDataFrame. \n",
```
Comment (tsnow03) [inline book/tutorials/SlideRule_applications/SlideRule_applications.ipynb:217]: We can inspect

File: book/tutorials/SlideRule_applications/SlideRule_applications.ipynb [hunk 4]
```diff
@@ -0,0 +1,1209 @@
...
+    "# concatenate the results into a single geodataframe\n",
+    "gdf3 = gpd.pd.concat(elevations)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Review GeoDataFrame output\n",
+    "Can inspect the columns, number of returns and returns at the top of the GeoDataFrame. There are different columns for the `atl08` request, which add the vegetation metrics.\n",
```
Comment (tsnow03) [inline book/tutorials/SlideRule_applications/SlideRule_applications.ipynb:973]: We can review

**Review by tsnow03 — APPROVED** [review]
Will want to render all the cells so the maps and things can be seen in the JupyterBook. May want to add a summary or wrap up but not required. Otherwise looks great!

**Pull request merged**